about-pses/example-pse.md

```diff
@@ -29,64 +29,189 @@ Create a function named `winner` that takes two arguments, the move for `player_
 
 <br>
 
-Even if we make a decision about how to handle a particular issue here, the unit tests may lead us to reconsider that decision. For example, if we decide to return `None` for invalid input, but the unit tests expect an error to be raised, we may need to change our implementation, but there's no need to come back and update the clarifying questions. The important thing is to think through the problem and make a decision based on the information we have at the time. When the tests differ from our initial thoughts, we can think of that as a case where the interviewer has given us new information that we need to incorporate into our solution.
+One of many possible sample responses could look like:
+
+<br>
+
+1. I need to implement a function called `winner` that takes two arguments: `player_1` and `player_2`.
+    - This means I need to create a function definition with two parameters named exactly `player_1` and `player_2`.
+
+2. Each argument represents a move for player_1 or player_2 and will be one of "rock", "paper", or "scissors".
+    - The values shown are strings, so I can assume the inputs will be strings holding one of these 3 values and that I will need to compare the strings in some way to find a winner.
+
+3. The function should return a string indicating the result of the game: "Player 1 wins!", "Player 2 wins!", or "It's a tie!"
+    - Aside from checking if someone wins, I need to consider the edge case where the players could tie.
+
+4. The second input/output example shows both players choosing "rock" and the result is a tie
+    - I am hypothesizing that two players choosing the same move results in a tie. 
+    - This is supported by the table of rules further below that shows the result as "It's a tie!" any time both players choose the same move.
+
+5. The table of rules contains text "input (first argument)" and "input (second argument)"
+    - This tells me when deciding who is the winner, I need to be mindful that the first argument's value always represents player_1 and the second argument represents player_2.
+    - This is supported by the first example where `player_1 = "rock"` and `player_2 = "scissors"`, and the result was `"Player 1 wins!"`.
+
+6. Based on the table of rules: "rock" beats "scissors", "scissors" beats "paper", "paper" beats "rock".
+    - When developing my solution, I will use these rules to guide the conditional statements I need to write to determine the winner.
+
+<br>
+
+We should keep in mind that sometimes, areas of clarification may not jump out at us when we first read the prompt. It's completely OK to think through the prompt, and even start working on it. As we start to write code, we may note something we missed before or realize that we need to ask a clarifying question. This is a normal part of the problem-solving process. So don't feel like we need to note everything or ask all our clarifying questions before we start working on the problem.
```

Comment:
```suggestion
We should keep in mind that sometimes, areas of clarification may not jump out at us when we first read the prompt. It's completely OK to think through the prompt, and even start working on it. As we start to write code, we may note something we missed before or realize that we need to ask a clarifying question. This is a normal part of the problem-solving process, so don't feel like we need to note everything or ask all our clarifying questions before we start working on the problem.
```

Reply:
This explanation is a rather long block of text and I feel like this is important to visually highlight this point. How do you feel about italicizing or bolding it?

```suggestion
We should keep in mind that sometimes, areas of clarification may not jump out at us when we first read the prompt. It's completely OK to think through the prompt, and even start working on it. **As we start to write code, we may note something we missed before or realize that we need to ask a clarifying question. This is a normal part of the problem-solving process. So don't feel like we need to note everything or ask all our clarifying questions before we start working on the problem.**
```

Reply:
To help break up the chunk of text, how do y'all feel about moving that last couple sentences to a bullet point? On re-reading, I also didn't love telling people "Don't feel a certain way!", and rephrased it a little. 

```suggestion
We should keep in mind that sometimes, areas of clarification may not jump out at us when we first read the prompt. It's completely OK to think through the prompt, and even start working on it. As we start to write code, we may note something we missed before or realize that we need to ask a clarifying question. 
- This is a normal part of the problem-solving process! We don't need to note everything or ask all our clarifying questions before we start working on the problem.
```

Reply:
Making it a bullet point looks good to me!